dask_geomodeling/tests/test_raster.py

```diff
@@ -1663,6 +1663,18 @@ def test_mask(self):
         self.assertEqual(view.get_data(**self.meta_request)["meta"], self.expected_meta)
         self.assertEqual(view.get_data(**self.time_request)["time"], self.expected_time)
 
+        # larger dtypes are supported too
+        view = raster.Mask(store=self.raster, value=1000)
+        data = view.get_data(**self.vals_request)
+        self.assertEqual(str(view.dtype), "uint16")
+        assert_equal(data["values"], 1000)
+
+        # larger dtypes are supported too
```

Comment:
The comment can have some work. Larger is a bit ambiguous (more bytes? higher values?), and it is the same as the above test.